### PR TITLE
Memoize false and nil values

### PR DESCRIPTION
**Pull request by benbalter**
/cc @spraints because Ruby. :smile:

### CODE REVIEW

**Comment by spraints** [comment]
:+1: Looks good!

File: lib/licensee.rb [hunk 1]
```diff
@@ -60,7 +60,7 @@ def confidence_threshold
     end
 
     def package_manager_files?
-      @package_manager_files ||= false
+      @package_manager_files || false
```
Comment (bkeepers) [inline lib/licensee.rb]: Does `@package_manager_files` ever get set?

Comment (benbalter) [inline lib/licensee.rb]: Package manager files is set via `Licensee.package_manager_files = true`. If the instance variable is true, it returns the instance variable. If the instance variable is `false`, it'd fall through the `||=` (as I learned yesterday) and would be `false` either way.

Comment (benbalter) [inline lib/licensee.rb]: Specifically `attr_writer :confidence_threshold, :package_manager_files`

Comment (bkeepers) [inline lib/licensee.rb]: Ah, thanks

**Pull request merged**